README.md

```diff
@@ -15,4 +15,25 @@ For now, two constructors are exposed by the TU module:
 Documentation
 =============
 
-Coming (not really) soon.
+TableFilter(tableElt, tolerance);
```

Comment:
Code content should be described as such, like this:

    ```js
    // This is an example
    console.log('Hello, World!');
    ```

This would make GitHub display it the clean way:

```js
// This is an example
console.log('Hello, World!');
```